### PR TITLE
Feature/bp cypress

### LINKED ISSUE #29: [🧑‍💻] Testing Home Page Flow : Page Load
😊  Should load a title and navigation
☹️ No sad path testing as your browser should be telling you that the internet is down!

### LINKED ISSUE #30: [🧑‍💻] Testing Home Page Flow : Page Navigation
😊 Should be able to click on navigation buttons to load other pages (spellbook, character classes)
😊 Should be able to click on navigation buttons and see the URL path change

### LINKED ISSUE #31: [🧑‍💻] Testing Character Class Page Flow : Home Page Button
😊 Should be able to click the 'Home' button to go back to the home page
😊 Should be able to click the 'Home' button and the URL path should go back to '/'

### LINKED ISSUE #33: [🧑‍💻] Testing Character Classes Page Flow : Character Class Links
😊 Should be able to click on a class name/icon to view spells for that class
😊 Should see an updated URL path that includes the name of the class

### LINKED ISSUE #34: [🧑‍💻] Testing A Character Class Page Flow : All Spells
😊 Should see all of the spells that can be used by the character class
☹️ Should see a message telling the user that the class can't use any spells (non-magical)

### LINKED ISSUE #35: [🧑‍💻] Testing A Character Class Page Flow : Navigating to Other Class
😊 Should be able to click on a different character class to view their spells
😊 Should see the URL path change to reflect character class change

### PR DESCRIPTION
**Pull request by bpeterson2579**
## [DnD Spells]

- [x] - Feature PR [🧩]
- [ ] - Refactor PR [⚙️]
- [ ] - BugFix PR [🐞]
- [ ] - User Interface PR [🖍]
- [ ] - Error Handling PR [⚠️]
- [x] - Testing PR [🧑‍💻]

**What (if any) features are you implementing?**

 - Added the README
 - Added Cypress testing
 - Closes #29 
 - Closes #30 
 - Closes #31 
 - Closes #33
 - Closes #34 
 - Closes #35 

**What (if anything) did you refactor?**

 - (NA)

**Were there any issues that arose?**

 - (NA)

**Is there anything needs to be prioritized/errors to be addressed?**

 - (NA)

**Goals? Next steps?**

 - (NA)

**Current Screenshots**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,57 @@
...
+- [Contributors](#Contributors)
+
+## Abstract
+Squizard's Spells is an app that allows its users to filter through Dungeons and Dragons 5th Edition spells and save those spells in a personal spellbook. Spells saved will persist on page load as well. When a user reaches the site, they will be able to look at their personal spellbook or browse spells. Then they can choose the character class that they are playing to narrow it down to spells that that class can use. Players can then click on spells to get the full details or bookmark them to save them to their spellbook.
+
+## Technologies
+-  HTML
+-  CSS
+-  JavaScript
+-  React
```
Comment (augustreid) [inline README.md:22]: Maybe could add React-Router, REST Api, Cypress

**Review by augustreid — APPROVED** [review]
Thanks, Brian! Looks good.

File: README.md (hunk 1 above)
Comment (elisebeall) [reply README.md:22]: Yes, definitely want to add those!  I'd also add some of the specific hooks like 'useReducer' & 'useContext'.  Could mention the custom hooks too 🤓 🤓

**Review by elisebeall — APPROVED** [review]
@bpeterson2579 few things that I think need to be added to the 'technologies' section README, but otherwise, great job!  Thanks!

**Pull request merged**